src/environment/firebase.env.js

```diff
@@ -0,0 +1,11 @@
+const firebaseConfig = {
+  apiKey: 'AIzaSyBzMz9H50Isa_z6jS558G7RxYGVrJ6pFFg',
+  authDomain: 'dims-53765.firebaseapp.com',
+  databaseURL: 'https://dims-53765-default-rtdb.europe-west1.firebasedatabase.app',
+  projectId: 'dims-53765',
+  storageBucket: 'dims-53765.appspot.com',
+  messagingSenderId: '78090258188',
+  appId: '1:78090258188:web:1650a2d996e9ca5bef2b29',
+};
```

Comment:
extract in `.env.local` or `.env` file and use them here, read about https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
I created .env file with config variables.
```suggestion
initializeApp({
  apiKey: process.env.REACT_APP_apiKey,
  authDomain: process.env.REACT_APP_authDomain,
  databaseURL: process.env.REACT_APP_databaseURL,
  projectId: process.env.REACT_APP_projectId,
  storageBucket: process.env.REACT_APP_storageBucket,
  messagingSenderId: process.env.REACT_APP_messagingSenderId,
  appId: process.env.REACT_APP_appId,
})
```